cmd/bdjuno/main.go

```diff
@@ -67,5 +71,18 @@ func getBasicManagers() []module.BasicManager {
 func getAddressesParser() messages.MessageAddressesParser {
 	return messages.JoinMessageParsers(
 		messages.CosmosMessageAddressesParser,
+		OsmoMessageAddressesParser,
 	)
 }
+
+// OsmoMessageAddressesParser returns the list of all the accounts involved in the given
+// message if it's related to the x/gamm module
+func OsmoMessageAddressesParser(_ codec.Codec, cosmosMsg sdk.Msg) ([]string, error) {
+	switch msg := cosmosMsg.(type) {
+
+	case *gammtypes.MsgCreateBalancerPool:
+		return []string{msg.Sender, msg.FuturePoolGovernor}, nil
+	default:
+		return []string{msg.String()}, nil
```

Comment:
This is not correct to be honest. We should just return an error instead since the `CosmosMessageAddressesParser` will take care of default messages instead 
```suggestion
		return []string{msg.Signer.String()}, nil
```